postgres-free-tier.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: 'The economics of a Postgres free tier'
+description: "Let's look at the numbers behind our free tier. How much does a database costs us and why are we giving them for free."
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/postgres-direct-access.jpg
+  alt: Postgres Platform Free Tier
+author: Tudor Golubenco
+date: 07-11-2024
+tags: ['engineering']
```

Comment:
Not sure if we also have a postgres tag?

Reply:
We have, it’s the one syndicated to Planet Postgres. But I was worrying that this might not respect their policy, so thought I don’t risk it.